.pre-commit-config.yaml

```diff
@@ -124,15 +124,15 @@ repos:
 # Use tools/codespell_ignore_lines_from_errors.py
 # to rebuild .codespell-ignore-lines
 - repo: https://github.com/codespell-project/codespell
-  rev: "v2.2.5"
+  rev: "v2.2.6"
   hooks:
   - id: codespell
     exclude: ".supp$"
-    args: ["-x.codespell-ignore-lines", "-Lccompiler"]
+    args: ["-x.codespell-ignore-lines", "-Lccompiler", "-w"]
```

Comment:
This could backfire (e.g. exceed line length limits, or even break something), I think it's better to not make the corrections automatic. It's pretty rare that it finds something and it's super quick to take care of manually.

Reply:
```suggestion
    args: ["-x.codespell-ignore-lines", "-Lccompiler"]
```

IMO, once it's working, it's much rarer to have it break on a newly added line, and you have to check auto-fixes anyway (black, Ruff, etc). What I keep seeing (in other projects too) is the above, where it adds a new check that catches a real misspelling, then I have to go in and manually fix it (I have ~5 projects today that need this).

But I'm fine to change it back to manual (it already did the fix I needed ;) ).

Reply:
> But I'm fine to change it back to manual

I'd prefer that, I volunteer to handle the fixes. (I believe that's less work (and less stressful) than dealing with surprises in the middle of other complex work.)

Reply:
If there were a lot of names, it might be higher risk, but code-spell is really good at avoiding false positives. Okay.

Reply:
(We can use the add `-w`, remove `-w` trick to fix them, too ;)